src/PKSim.Core/Mappers/MoleculeToQueryExpressionSettingsMapper.cs

```diff
@@ -28,6 +30,17 @@ public QueryExpressionSettings MapFrom(IndividualMolecule molecule, ISimulationS
          return new QueryExpressionSettings(expressionContainer, molecule.QueryConfiguration, moleculeName);
       }
 
+      public QueryExpressionSettings MapFrom(ExpressionProfileBuildingBlock expressionProfileBuildingBlock)
+      {
+         var containerGroups = expressionProfileBuildingBlock.
+            Where(x => x.Value.HasValue && x.HasExpressionName()).
```

Comment:
value is a nullable?

Reply:
Yes, it's a PathAndValueEntity like start value